readarr/faq.md

```diff
@@ -53,9 +55,10 @@ dateCreated: 2021-05-25T20:01:09.320Z
   - [How do I stop the browser from launching on startup?](#how-do-i-stop-the-browser-from-launching-on-startup)
   - [Weird UI Issues](#weird-ui-issues)
   - [VPNs, Jackett, and the \*ARRs](#vpns-jackett-and-the-arrs)
+    - [Use of a VPN](#use-of-a-vpn)
   - [Jackett's /all Endpoint](#jacketts-all-endpoint)
     - [Jackett /All Solutions](#jackett-all-solutions)
-  - [Why are there two files? | Why is there a file left in downloads?](#why-are-there-two-files-why-is-there-a-file-left-in-downloads)
+  - [Why are there two files? | Why is there a file left in downloads?](#why-are-there-two-files--why-is-there-a-file-left-in-downloads)
```

Comment:
Need to manually fix these. Wikijs ignores the special chars `--` to `-`

Reply:
I should learn more about Wikijs. `:shame:`

Reply:
Learn by doing (it was something I stumbled on as well)